src/components/Header.js

```diff
@@ -1,10 +1,73 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
-}
+const Header = ({ userList, toggleUser }) => {
+    const [userName, setUserName] = useState('고은');
+    const [userStatus, setUserStatus] = useState('현재 활동중');
+    const [userImage, setUserImage] = useState(userList[1]);
+
+    const onClick = () => {
+        if (toggleUser()) {
+            setUserImage(userList[1]);
+            setUserName('고은');
+        } else {
+            setUserImage(userList[0]);
+            setUserName('정쿨');
```

Comment:
여기서 중복되는 코드가 있는데, 저라면 이렇게 함수를 만들어서 사용할 것 같습니다!!
```suggestion
const setUser = (userName, userImage) => {
            setUserImage(useImage);
            setUserName(userName);
            }
```

Reply:
저렇게 함수를 만들어 놓으면 볼 때도 더 편할거 같네요! 수정하겠습니다~